app.py

```diff
@@ -1,41 +1,54 @@
+from _pytest.assertion import pytest_assertrepr_compare
 from flask import Flask, render_template, request, redirect, url_for
 import ApiAccess as api
+import View_Model as model
 
 
-app = Flask(__name__)
-app.config.from_object('flask_config.Config')
+def create_app():
```

Comment:
If we're using the create_app syntax, you should be setting all of the routes before you return the app; otherwise (as it currently stands) anyone who calls this function gets an app without any endpoints configured; only your `app` object has those set